BuildResidentialHPXML/measure.rb

```diff
@@ -345,6 +345,13 @@ def arguments(model)
     arg.setDefaultValue(0.0)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('geometry_rim_joist_height', true)
+    arg.setDisplayName('Geometry: Rim Joist Height')
+    arg.setUnits('ft')
+    arg.setDescription("The height of the rim joists. Only applies to basements/crawlspaces. Only applies to #{HPXML::ResidentialTypeSFD} and #{HPXML::ResidentialTypeSFA} units.")
```

Comment:
Why does this not apply to MF units? Your other new argument doesn't have a similar statement, and I would think that it should apply to a MF unit over a crawlspace.

Reply:
You're right. I think I forgot to handle MF because there are no test xml files with units adjacent to foundations.